content/en/docs/compatibility/_index.md

```diff
@@ -15,6 +15,7 @@ See the following list of compatible Kubernetes distributions and platforms for
  - [Digital Ocean Kubernetes (DOKS)](https://www.digitalocean.com/products/kubernetes)
  - [Google Kubernetes Engine (GKE)](https://cloud.google.com/kubernetes-engine)
  - [k3d](https://k3d.io)
+ - [minikube](https://minikube.sigs.k8s.io/docs/) (explicitly pass `--container-runtime=containerd` and ensure you're on version `>= 1.33`)
```

Comment:
```suggestion
 - [minikube](https://minikube.sigs.k8s.io/docs/) (explicitly pass `--container-runtime=containerd` and ensure you're on containerd version `>= 1.33`)
```

Reply:
this should be minikube version >= 1.33

Reply:
🙈 (you're right - maybe we should reformat this to be a little clearer, given that it was easy to misread 😅)

Reply:
updated to minkube! thanks